contracts/Colony.sol

```diff
@@ -46,10 +63,18 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     colonyNetworkAddress = _address;
 
     // Initialise the task update reviewers
-    setFunctionReviewers(0xda4db249, 0, 2); // setTaskBrief => manager, worker
-    setFunctionReviewers(0xcae960fe, 0, 2); // setTaskDueDate => manager, worker
-    setFunctionReviewers(0x6fb0794f, 0, 1); // setTaskEvaluatorPayout => manager, evaluator
-    setFunctionReviewers(0x2cf62b39, 0, 2); // setTaskWorkerPayout => manager, worker
+    setFunctionReviewers(bytes4(keccak256("setTaskBrief(uint256,bytes32)")), MANAGER, WORKER);
+    setFunctionReviewers(bytes4(keccak256("setTaskDueDate(uint256,uint256)")), MANAGER, WORKER);
+    setFunctionReviewers(bytes4(keccak256("setTaskSkill(uint256,uint256)")), MANAGER, WORKER);
+    setFunctionReviewers(bytes4(keccak256("setTaskManagerPayout(uint256,address,uint256)")), MANAGER, MANAGER);
```

Comment:
This was change from last review. Alex thinks that we should be consistent with these task change calls. It will be easier for the front-end.

Reply:
Hm, this doesn't really solve the inconsistency issue if we need to double-sign from the manager.

Reply:
From the client it can be signed only by manager. There is a [check](https://github.com/JoinColony/colonyNetwork/blob/develop/contracts/ColonyTask.sol#L170) in `executeTaskChange` that will solve this. Its a little hacky maybe, but it works.

Reply:
Ah, I hadn't clocked that check would also catch this, but you're right, it does. Can we add an additional comment there? The comment currently only talks about both roles being assigned to the same user, which while it is sort-of what is happening here, is quite misleading.